avaframe/tests/test_geoTrans.py

```diff
@@ -18,7 +18,8 @@ def test_projectOnRaster(capfd):
 
     rasterdata = np.array(([0, 1, 2], [2, 0.5, 5]))
     Points['x'] = np.array((-5, -0.000000001, -5, 2, 0, -2.5, -2, 2))
-    Points['y'] = np.array((2, 6.99999999999, 6.999999999999, 6, 10, 4.5, np.nan, 8))
+    Points['y'] = np.array((2, 6.99999999999, 6.999999999999, 6, 10, 4.5,
```

Comment:
It should be possible to use 7.0 and 0 and 2 an such, since this is only a computational artifact...

Reply:
that is actually my input data, I want it to be like this